src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs

```diff
@@ -177,7 +177,7 @@ public void reader_14()
             ReadFrom(xml);
         }
 
-        //XElement: non-wellformed xml
+        //XElement: non�well-formed xml
```

Comment:
I think I typed it as non–well-formed following the rule that an en-dash should be used when creating a compound from a compound, and it seems the dash got eaten in the save. A question therefore is, why didn't I get a warning on that?